nix/pgadmin.nix

```diff
@@ -118,7 +118,7 @@ in
                       ${config.package}/bin/.pgadmin4-setup-wrapped setup
                     else
                       # pgadmin-8.2 has .pgadmin4-cli-wrapped
-                      ${config.package}/bin/.pgadmin4-cli-wrapped setup
+                      ${config.package}/bin/.pgadmin4-cli-wrapped setup-db
```

Comment:
What's the context for this change? Is services-flake not supported on nixpkgs older to this update anymore?

Reply:
in the PR #115, i made a mistake with the setup command for pgadmin 8.2. since the older nixpkgs had pgadmin 7.5, i could not test this and it got missed.

for older nixpkgs, pgadmin has `pgadmin4-setup` executable to setup the configuration db, and for newer nixpkgs, it has `pgadmin4-cli` executable to setup the db. so services-flake would support both

```sh
  if [ -f ${config.package}/bin/.pgadmin4-setup-wrapped ]; then
    # pgadmin-7.5 has .pgadmin4-setup-wrapped
    ${config.package}/bin/.pgadmin4-setup-wrapped setup
  else
    # pgadmin-8.2 has .pgadmin4-cli-wrapped
    ${config.package}/bin/.pgadmin4-cli-wrapped setup-db
  fi
```